cobalt/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -144,7 +148,18 @@ public StarboardBridge(
 
   private native void closeNativeStarboard(long nativeApp);
 
-  private native long nativeCurrentMonotonicTime();
+  @NativeMethods
+    interface Natives {
+        void onStop();
+
+        long currentMonotonicTime();
+
+        // boolean initJNI();
+
+        // long startNativeStarboard();
+
+        // void closeNativeStarboard(long nativeApp);
```

Comment:
What's the plan for these three, are they commented out on purpose? Would any functionality be broken? Assuming yes and no, I'd suggest a `// TODO(372559388)` explaining the plan.